docker-images/kafka/scripts/kafka_connect_config_generator.sh

```diff
@@ -17,6 +17,9 @@ producer.ssl.truststore.password=${CERTS_STORE_PASSWORD}
 
 consumer.ssl.truststore.location=/tmp/kafka/cluster.truststore.p12
 consumer.ssl.truststore.password=${CERTS_STORE_PASSWORD}
+
+admin.ssl.truststore.location=/tmp/kafka/cluster.truststore.p12
+admin.ssl.truststore.password=${CERTS_STORE_PASSWORD}
```

Comment:
because we are adding TLS configuration for `admin` as we did for `producer` and `consumer`, aren't useless now the lines above without prefixes (11 to 13)? Or at least, only 13 is useful to set the PKCS12 type unless we set them for all the prefixed properties as we do for the following keystores.

Reply:
TBH, I didn't found any docs specifying how it behaves etc. The non-prefixed lines seem to be used by Connect framework it self - to create the toppic it needs etc. The `admin.` prefixed lines seened to be used by the connectors for admin client operations. So we should need both of them.

Reply:
my understanding was that the properties without prefixes were somehow shared by the corresponding properties with prefix in order to avoid duplication.

Reply:
at this point why don't we have the PKCS12 type set for them (I mean `consumer.ssl.truststore`, `admin.ssl.truststore` etc etc)?

Reply:
That was my original understanding as well, but it is not the case - at least not for all of them. Otherwise we would have just those without prefix. As for the store type, it actually works without it - it seems to  be able to handle both PKCS12 as well as JKS without it being specified.